.travis.yml

```diff
@@ -61,7 +61,14 @@ jobs:
         SDK=python
         SDK_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
         FULLSTACK_TEST_REPO=ProdTesting
-
+    - stage: 'Test'
+      python: "pypy"
+      before_install:
+        - pip install "cryptography>=1.3.4,<=3.1.1" # installing in before_install doesn't re-install a latest version of the same package in the next stage.
```

Comment:
Can you fix grammatical error: Change "a latest version of the same" to "the latest version of the same".

Reply:
done